static/src/javascripts/projects/commercial/modules/messenger.ts

```diff
@@ -44,7 +44,7 @@ type ListenerCallback = (
 	 * predict what the iframe will send. It is the responsibility of the callback
 	 * to obtain a value of the desired type (or fail gracefully)
 	 */
-	specs: unknown | null | undefined,
+	specs: unknown | undefined,
```

Comment:
```suggestion
	specs?: unknown,
```

Reply:
I'm not sure this will work since it's followed by a non-optional arg